src/index.js

```diff
@@ -52,6 +52,7 @@ app.use('/health', (req, res) => {
 // Routes
 app.use('/user', routes.user);
 app.use('/userRole', routes.userRole);
+app.use('/contact', routes.contact);
```

Comment:
This method of adding routes is outdated. I've made the routing more dynamic so this type of addition shouldn't be needed anymore.